audience_manager/index.rst

```diff
@@ -0,0 +1,14 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO Audience Manager documentation!
```

Comment:
This is title of this section of documentation. In this case it's Audience manager app. You can also add short description of the app before `toctree`.